src/inject/dynamic-theme/modify-css.ts

```diff
@@ -85,9 +85,26 @@ export function getModifiedUserAgentStyle(theme: Theme, isIFrame: boolean, style
     lines.push(`    border-color: ${modifyBorderColor({r: 76, g: 76, b: 76}, theme)};`);
     lines.push(`    color: ${modifyForegroundColor({r: 0, g: 0, b: 0}, theme)};`);
     lines.push('}');
-    lines.push('a {');
-    lines.push(`    color: ${modifyForegroundColor({r: 0, g: 64, b: 255}, theme)};`);
-    lines.push('}');
+    let linkColor: string;
+    let visitedLinkColor: string;
+    if (theme.linkColor === 'auto') {
+        linkColor = modifyForegroundColor({r: 0, g: 64, b: 255}, theme);
+    } else {
+        const hsl = rgbToHSL(parse(theme.linkColor));
+        const lighten = (lighter: number) => ({...hsl, l: clamp(hsl.l + lighter, 0, 1)});
+        linkColor = hslToString(hsl);
+        visitedLinkColor = hslToString(lighten(0.2));
```

Comment:
I thought you will add another color picker for visited links, but let's try a single one. Visited links usually differ by color (usually violet), so let's try to do lightness +0.2, saturation -0.3 and hue +45 (or between 30 and 45 if it gets too red). If hue surpasses 360, it should start from zero (subtract 360).